src/Blendedness.cc

```diff
@@ -284,6 +279,22 @@ BlendednessAlgorithm::BlendednessAlgorithm(Control const & ctrl,  std::string co
     }
 }
 
+float BlendednessAlgorithm::computeAbsExpectation(float data, float variance) {
+    float normalization = 0.5f*boost::math::erfc(-data/std::sqrt(2.0f*variance));
```

Comment:
It hadn't occurred to me to check, but it looks like it is.  I'll update all the code using the boost version in this file.